components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/base/includes/activities_info.html

```diff
@@ -77,10 +77,16 @@
                 new_results = -1;   // hide notification number
             }
             OME.displayStatus(new_results, inprogress);
+            if (recursive) {
+                setTimeout(function(){
+                    OME.activitiesUpdate(true);
+                }, 5000);
+            }
 
         }).error(function() {
```

Comment:
Also https://api.jquery.com/jquery.get/ The jqXHR.success(), jqXHR.error(), and jqXHR.complete() callback methods introduced in jQuery 1.5 are deprecated as of jQuery 1.8 could you start using new one please?